CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to 7 days to die sever manager
+You can find all the information about the server [here](https://docs.csmm.app/en/7D2D)
+## Thanks for taking the time to contribute
+The following is a set of guidelines for contributing to 7 days to die sever manager. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Links to some resources
+
+You can join the discord server to directly communicate with the developers by clicking [here](http://catalysm.net/discord).
+
+You can find all information about the CSMM [here](https://docs.csmm.app/).
+
+You can find the installation guide [here](https://docs.csmm.app/en/CSMM/self-host/installation.html).
+
+
+
+## Code of Conduct for Contributors
+This project and everyone participating in it is governed by the code of conduct for contributors which you can find [here](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+
+## How can I contribute?
+1. You can report bugs/glitches/faults that you are likely to come across when using the project server and client apps.
+2. You can suggest enhancements that you would want to see implemented on the project - which gets voted in the roadmap.
```

Comment:
we have a roadmap?
if so, can you link it
if not, can you remove this?

Reply:
I'll remove it that part.

Instead should I add you can suggests it, by creating an issue ?

Reply:
Yep, the way to suggest new features is with issues. Good addition!